src/kem/bike/additional_r3/noop_main.c

```diff
@@ -5,6 +5,6 @@
  * AWS Cryptographic Algorithms Group.
  */
 
-int main() {
+int main(void) {
```

Comment:
This change cannot possibly have been subject to this statement: 

> [I've added the void parameter to these templates](https://github.com/open-quantum-safe/liboqs/pull/1293/commits/78fcd40105b4d093a3b898acffdd009f0ca73114) 
> And everything was fixed.

So I wonder: How much of this PR is generated from the changes to `kem_scheme.c` and `sig_scheme.c` and how much is manual?

Reply:
For frodokem, bike and picnic, changes were manual. For any other `kem` and `sig`, changes were generated by the script `copy_from_upstream.py` in [this commit](https://github.com/open-quantum-safe/liboqs/pull/1293/commits/f6d7b998b2dfdafc2cbe9712fa4c06123d4684f5)

Reply:
Thanks for the explanation -- I'd then propose to merge as-is. 

@dstebila @christianpaquin : Please check whether you'd want to bring these changes upstream yourself or ask @thb-sb for such contribution.

Still not entirely happy about dropping the warnings about some algorithm irregularities, though. Thoughts welcome.

Reply:
> @dstebila @christianpaquin : Please check whether you'd want to bring these changes upstream yourself or ask @thb-sb for such contribution.

It would be good to get these changes into the Frodo upstream; we don't need to wait for this, we can push them there afterwards.

> Still not entirely happy about dropping the warnings about some algorithm irregularities, though. Thoughts welcome.

Are you referring to the warnings in the `docs/algorithms` files? I don't understand why they've been removed either. If some texts need to be changed, I'd suggest doing it in a different PR to limit the scope of this one.

Reply:
> It would be good to get these changes into the Frodo upstream; we don't need to wait for this, we can push them there afterwards.

OK, then those changes are good, too.

> Are you referring to the warnings in the docs/algorithms files? I don't understand why they've been removed either. If some texts need to be changed, I'd suggest doing it in a different PR to limit the scope of this one.

The logic for this happening is clear: As soon as the `copy_from_upstream` script runs, those manual warnings disappear. The question is whether we should (manually) retain/reinstate them or whether we should now "move" on them (removing Rainbow for good, say). If the latter, this PR truly can be merged and I volunteer to do a PR to remove Rainbow :-)

Reply:
> > @dstebila @christianpaquin : Please check whether you'd want to bring these changes upstream yourself or ask @thb-sb for such contribution.
> 
> It would be good to get these changes into the Frodo upstream; we don't need to wait for this, we can push them there afterwards.

Agreed, the link between FrodoKEM upstream and liboqs is manually maintained, not automated, so fine to go in either direction.